db/seeds.rb

```diff
@@ -55,3 +55,9 @@
 
   Post.insert_all(posts)
 end
+
+sample_size = Post.count * 0.7
+Post.all.sample(sample_size).each do |post|
+  category = Category.all.sample
+  post.categories << category
+end
```

Comment:
It's hard to mess around with `post`s and `category`s if none of your seeded posts have categories. 😆 So I fixed it.